### DISCUSSION TITLE
Can't capitalize inline cross-references

### DISCUSSION THREAD

**Discussion by arnaudgallou** [quarto-dev/quarto-cli#1112] [Q&A] [answered]
I can't manage to capitalize a cross-reference by changing the syntax of the inline reference as explained in the [documentation](https://quarto.org/docs/authoring/cross-references.html#references).

Below is an [example](https://github.com/arnaudgallou/dummy-qmd/blob/main/crossrefs/crossrefs.qmd) with the resulting output being visible [here](https://github.com/arnaudgallou/dummy-qmd/blob/main/crossrefs/crossrefs.md).

```
---
title: Cross references
editor_options:
  markdown: 
    wrap: sentence
format: gfm
crossref: 
  title-delim: .
  fig-title: Fig.
  fig-prefix: fig.
  eq-prefix: eq.
fig-format: svg
---

## Figures

{{< include ../_figure.qmd >}}

**single plot**

@fig-foo

@Fig-foo

**faceted plot**

[@fig-foo]A

**supplementary plot**

Fig. S[-@fig-foo]

## Equations

$$\alpha=2$$ {#eq-a}

@eq-a

@Eq-a
```

I'm using Quarto v. 0.9.549 on RStudio v. 4.2.0, running on macOS Monterey.

**Comment by cderv** [comment, marked as answer]
I think this is an issue so I created #1113 to follow up

**Reply by cderv** [reply]
This should now be fixed

**Reply by arnaudgallou** [reply]
I confirm that it works now.
